probe/probe.go

```diff
@@ -534,8 +534,9 @@ func getPostureData(probeData []KubeArmorProbeData) map[string]string {
 // sudo systemctl status kubearmor
 func isSystemdMode() bool {
 	cmd := exec.Command("systemctl", "status", "kubearmor")
-	_, err := cmd.CombinedOutput()
+	out, err := cmd.CombinedOutput()
 	if err != nil {
+		log.Println("systemctl status kubearmor cannot be executed:", string(out))
```

Comment:
We should refactor the function to return error, and use Cobra Error handler to capture the errors. That way we will not show the error incase we find KubeArmor in kubernetes